log

```diff
@@ -0,0 +1,343 @@
+--- FAIL: TestPrivateWeave (0.88s)
+	integration_test.go:181: diff:
+		+ resource "aws_autoscaling_attachment" "bastions-bastions-privateweave-example-com" {
+		- resource "aws_autoscaling_attachment" "bastion-privateweave-example-com" {
+		    elb = "${aws_elb.bastion-privateweave-example-com.id}"
+		+   autoscaling_group_name = "${aws_autoscaling_group.bastions-bastions-privateweave-example-com.id}"
+		-   autoscaling_group_name = "${aws_autoscaling_group.bastion-privateweave-example-com.id}"
+		  }
+		  
+		...
+		  }
+		  
+		+ resource "aws_autoscaling_group" "bastions-bastions-privateweave-example-com" {
+		- resource "aws_autoscaling_group" "bastion-privateweave-example-com" {
+		+   name = "bastions.bastions.privateweave.example.com"
```

Comment:
### Regarding IG's called `bastions` (pluralized)

Do we have a standard here for singular/plural IG's? I vaguely remember `bastion` floating around in some other code..